cpp/include/cudf/io/experimental/hybrid_scan.hpp

```diff
@@ -499,6 +499,31 @@ class hybrid_scan_reader {
     parquet_reader_options const& options,
     rmm::cuda_stream_view stream) const;
 
+  /**
+   * @brief Get byte ranges of column chunks of all (or selected) columns
+   *
+   * @param row_group_indices Input row groups indices
+   * @param options Parquet reader options
+   * @return Vector of byte ranges to column chunks of all (or selected) columns
+   */
+  [[nodiscard]] std::vector<byte_range_info> all_column_chunks_byte_ranges(
+    cudf::host_span<size_type const> row_group_indices,
```

Comment:
Can this use `std::span`? We'd like to use `std::span` instead of our custom spans when possible.
https://github.com/rapidsai/cudf/issues/20539

Reply:
Unfortunately it's also a part of the same API series using `host_span`s. I am gonna replace all of them at once in a PR soon.

Reply:
should we have an implicit `host_span` conversion from `std::span`?